webui/module/Restore/view/restore/restore/index.phtml

```diff
@@ -57,36 +58,37 @@ $this->headTitle($title);
 </div>
 <!-- modal-001 end -->
 
-<!-- modal-002 start -->
+<!-- modal-002 start (confirmation about started restore job) -->
 <div id="modal-002" class="modal fade modal-002" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel2">
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
-   <div class="modal-header">
-   <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-   <h4 class="modal-title" id="myModalLabel"><?php echo $this->translate("Director message"); ?></h4>
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel"><?php echo $this->translate("Director message"); ?></h4>
       </div>
       <div class="modal-body">
          <pre><code>
 <?php echo $this->result; ?>
          </code></pre>
       </div>
       <div class="modal-footer">
-         <?php $jobid = substr($this->result, stripos($this->result, 'JobId=') + 6 ); ?>
-         <a class="btn btn-primary" href="<?php echo $this->basePath() . '/job/details/' . $jobid; ?>"><?php echo $this->translate("Job report"); ?></a>
+         <?php if (isset($this->restore_jobid)): ?>
+            <a class="btn btn-primary" href="<?php echo $this->basePath() . '/job/details/' . $this->restore_jobid; ?>"><?php echo $this->translate("Job report"); ?></a>.
+         <?php endif; ?>
          <button type="button" class="btn btn-primary" data-dismiss="modal"><?php echo $this->translate("Close"); ?></button>
```

Comment:
I feel like this should be a default button so as to not have 2 primary buttons on the same form. What do you think ?